docs/OCP_CI_Tutorials/Containers/Container_Creation_Guide.md

```diff
@@ -11,16 +12,19 @@
     - [Execute in the Container](#execute-in-the-container)
 
 ## Introduction
+
 Prior to discussing the how to use containers in OpenShift CI, it is important to understand the basic structure of a CSPI test in OpenShift CI. 
```

Comment:
```suggestion
Before we discuss how to use containers in OpenShift CI, it is important to understand the basic structure of a CSPI test in OpenShift CI. 
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)